src/engine/Server.cpp

```diff
@@ -578,6 +613,11 @@ boost::asio::awaitable<void> Server::processQuery(
     LOG(INFO) << "Requested media type of result is \""
               << ad_utility::toString(mediaType.value()) << "\"" << std::endl;
 
+    ad_utility::websocket::WebSocketNotifier webSocketNotifier{
+        getQueryId(request), webSocketManager};
+    auto notifierFunction = [&webSocketNotifier](std::string json) {
+      webSocketNotifier(std::move(json));
+    };
```

Comment:
@joka921 I was thinking about making this an implicit conversion, but in the end I decided against it because this way you can clearly see that a "conversion" just provides a different view on the whole thing via reference. Thoughts?